.github/workflows/update_lock_file.yml

```diff
@@ -0,0 +1,41 @@
+name: Update lock file
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    # once a month
+    - cron: 0 5 25 * *
+
+jobs:
+  pixi-update:
+    runs-on: ubuntu-latest
```

Comment:
We need `if: github.repository == 'scipy/scipy'` here to prevent runs being triggered on forks.